activerecord/lib/active_record/insert_all.rb

```diff
@@ -88,6 +90,19 @@ def keys_including_timestamps
     private
       attr_reader :scope_attributes
 
+      def resolve_attribute_aliases
```

Comment:
Do you think we could have a fast path where we check if the first record contain any aliases, and if not we just skip that method?

Reply:
Yes, sure, good idea 👍